_site/index.html

```diff
@@ -92,30 +92,26 @@ <h3>
             Who?
           </h3>
           <p>
-            The specification is standardized by the TC39 (Technical committee 39) of ECMA (European
-            Computer Manufacturers Association). You can read our <a href="https://ecma-international.org/memento/TC39.htm#Scope">the scope of our work here</a>
+            TC39 is a body of JS developers, implementers, academics and more, collaborating with the community to maintain and evolve the definition of JS. We are part of <a href="http://www.ecma-international.org/">Ecma International</a>.
```

Comment:
As a non native English speaker; I don't know what "body of JS developers" means. Could we rephrase it to group? or something equivalent.

Reply:
This needs an Oxford comma after “academics”

Reply:
@xtuc Would the word "group" make more sense?

@ljharb I didn't realize all TC39 web properties needed to follow UK English style. Is that the case? Do you think we need to use UK spellings?

Reply:
The Oxford comma isn't a UK style, it's an unambiguous style used throughout the US - one that is used elsewhere in the document.

Reply:
An amusing US court case where the lack of an Oxford comma cost a company millions: https://www.nytimes.com/2018/02/09/us/oxford-comma-maine.html

Reply:
Wow, that missing comma is definitely not an Oxford comma. Anyway, I'll insert the commas here.